### PR TITLE
Fix #84 add set module types function

### LINKED ISSUE #84: Feature Request: Add Set-ModuleTypes
Much like you have the `Set-ModuleFunctions` and `Set-ModuleFormats` functions, it would also be nice to have a `Set-ModuleTypes` function for updating the `TypesToProcess` array in the module manifest.

### PR DESCRIPTION
**Pull request by fourpastmidnight**
Fixes #84.

Adds a `Set-ModuleTypes` function, much like the existing `Set-ModuleFunctions`, `Set-ModuleAliases`, and `Set-ModuleFormats` functions.

### CODE REVIEW

File: BuildHelpers/Public/Set-ModuleTypes.ps1 [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        $ModulePSD1Path = Join-Path $Parent $File
+        if(-not (Test-Path $ModulePSD1Path))
+        {
+            Throw "Could not find expected module manifest '$ModulePSD1Path'"
+        }
+
+        if(-not $TypesToProcess)
+        {
+            $TypesPath = Join-Path -Path $Parent -ChildPath $TypesRelativePath
+            $TypesList = Get-ChildItem -Path $TypesPath\*.ps1xml
```
Comment (RamblingCookieMonster) [inline BuildHelpers/Public/Set-ModuleTypes.ps1]: Might be nitpicking but I like to use `/` nowadays just to cover xplat stuff if it's ever needed/possible, if that makes sense?

**Comment by RamblingCookieMonster** [comment]
Hiyo!

Awesome idea!  LGTM, left one maybe-nitpicky note.  Not a show stopper - will merge this over the weekend, if you change it awesome, if not I'll just change it some other time, either works - thanks!

**Comment by fourpastmidnight** [comment]
No worries. If you like, I can update them in `Set-ModuleFormats`, too.

Regards,

Craig

Sent from my Windows 10 phone

File: BuildHelpers/Public/Set-ModuleTypes.ps1 (hunk 1 above)
Comment (fourpastmidnight) [reply BuildHelpers/Public/Set-ModuleTypes.ps1]: Sounds good . I just updated this in the latest push to the branch. I also took the liberty to udpate `Set-ModuleFormats` while I was at it.

**Comment by RamblingCookieMonster** [comment]
Wheee!  Alrighty, then, merging!

**Pull request merged**